hooks/test_post_gen_project.py

```diff
@@ -49,3 +56,57 @@ def test_copy_journal_template_files_bad(
         match=errormessage,
     ):
         copy_journal_template_files(input, project_dir)
+
+
+# C1: There are bib files in the cloned directory.
+#   Expect all bib name are inserted into manuscript,
+#   and bib files are copied into manuscript's parent dir
+#   exists.
+# C2: There are no bib files in the cloned directory.
```

Comment:
I think this would be safer in a mark.parameterize.  Maybe have a list of missing files as you did before and unlink all the files in the list.  Make it an empty list for the first one and have the source bib files for the second one.